semeio/workflows/misfit_preprocessor/config.py

```diff
@@ -251,51 +252,40 @@ def _inject_default_t(config):
             cs.MetaKeys.ElementValidators: (_one_of(SPEARMAN_CORRELATION, AUTO_SCALE),),
             cs.MetaKeys.Default: AUTO_SCALE,
         },
-        SPEARMAN_CORRELATION: SPEARMAN_CORRELATION_SCHEMA,
-        AUTO_SCALE: AUTO_SCALE_SCHEMA,
+        "clustering": HIERARCHICAL_SCHEMA,
+        "pca": PCA_SCHEMA,
     },
 }
 
 
-_SCALING_SCHEMA = {
+@cs.transformation_msg("Detecting method.")
```

Comment:
Maybe improve this message a bit?

Reply:
Would `Checking workflow schema in the configuration file` be better?

Reply:
We just made it!